dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/LinkageCheckerArguments.java

```diff
@@ -170,6 +170,14 @@ private static Options configureOptions() {
             .build();
     options.addOption(exclusionFile);
 
+    Option writeAsExclusionFile =
+        Option.builder("w")
+            .longOpt("write-as-exclusion-file")
```

Comment:
exclusion-file

Reply:
This option is different from specifying exclusion file to filter linkage errors. Did you mean something further than that?

Reply:
OK. Didn't realize we were already using exclusion-file. Still write-as-exclusion-file isn't clear. Other names?